doc/tutorials/enable_laag_secure_boot.rst

```diff
@@ -0,0 +1,276 @@
+
+   |
+
+   .. figure:: images/enroll_pk_key_6.png
+
+   |
+
+#. Press :kbd:`ESC` to go back to the **Secure Boot Configuration** interface.
+
+   Now the **Current Secure Boot State** is **Enabled** and **Attempt Secure Boot** option is selected.
+
+   .. figure:: images/secure_boot_enabled.png
+
+   |
+
+#. Go back to UEFI GUI main interface and select **Reset** to perform a formal
+   reset/shutdown to ensure the key enrollment is taking effect in the next boot.
+
+   .. figure:: images/reset_in_bios.png
+
+   |
+
+#. Type ``reset -s`` to shutdown the guest in the UEFI shell.
+
+   .. figure:: images/reset_in_uefi_shell.png
+
+   |
+
+Sign the Clear Linux image
+**************************
+
+Follow these commands to sign the Clear Linux VM binaries.
+
+#. Download and decompress the Clear Linux image::
+
+   $ wget https://download.clearlinux.org/releases/31080/clear/clear-31080-kvm.img.xz
+   $ unxz clear-31080-kvm.img.xz
+
+#. Download the script to sign image::
+
+   $ wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/scripts/sign_image.sh
+
+#. Run the script to sign image.
+
+   .. code-block:: none
+
+      $ sudo sh sign_image.sh clear-31080-kvm.img db.key db.crt
+      /mnt/EFI/BOOT/BOOTX64.EFI
+      warning: data remaining[93184 vs 105830]: gaps between PE/COFF sections?
+      warning: data remaining[93184 vs 105832]: gaps between PE/COFF sections?
+      Signing Unsigned original image
+      sign /mnt/EFI/BOOT/BOOTX64.EFI succeed
+      /mnt/EFI/org.clearlinux/bootloaderx64.efi
+      warning: data remaining[1065472 vs 1196031]: gaps between PE/COFF sections?
+      warning: data remaining[1065472 vs 1196032]: gaps between PE/COFF sections?
+      Signing Unsigned original image
+      sign /mnt/EFI/org.clearlinux/bootloaderx64.efi succeed
+      /mnt/EFI/org.clearlinux/kernel-org.clearlinux.kvm.5.2.17-389
+      Signing Unsigned original image
+      sign /mnt/EFI/org.clearlinux/kernel-org.clearlinux.kvm.5.2.17-389 succeed
+      /mnt/EFI/org.clearlinux/loaderx64.efi
+      warning: data remaining[93184 vs 105830]: gaps between PE/COFF sections?
+      warning: data remaining[93184 vs 105832]: gaps between PE/COFF sections?
+      Signing Unsigned original image
+      sign /mnt/EFI/org.clearlinux/loaderx64.efi succeed
+
+#. You will get the signed Clear Linux image: ``clear-31080-kvm.img.signed``
+
+Boot Clear Linux signed image
+*****************************
+
+#. Modify the ``launch_uos.sh`` script to use signed image.
+
+   .. code-block:: none
+      :emphasize-lines: 5,6,8
+   
+      $ sudo vim /usr/share/acrn/samples/nuc/launch_uos.sh
+   
+      acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge \
+        -s 2,pci-gvt -G "$3" \
+        -l com1,stdio \
+        -s 5,virtio-console,@pty:pty_port \
+        -s 6,virtio-hyper_dmabuf \
+        -s 3,virtio-blk,./clear-31080-kvm.img.signed \
+        -s 4,virtio-net,tap0 \
+        -s 7,virtio-rnd \
+        --ovmf w,/usr/share/acrn/bios/OVMF.fd \
```

Comment:
After key enrollment, the "w" (writeback) option is optional. Users can leave it in the script if they want other EFI changes to be persistent. If the "w" option is removed, EFI will reset upon resets/shutdowns.

Reply:
Thanks for the clarification, removed ``w,`` option in this section.